src/components/box/box.component.tsx

```diff
@@ -23,6 +24,9 @@ export type BoxSizing = "content-box" | "border-box";
 export type AllowedNumericalValues = typeof GAP_VALUES[number];
 export type Gap = AllowedNumericalValues | string;
 
+type DesignTokensType = keyof typeof DesignTokens;
+export type BoxShadowsType = Extract<DesignTokensType, `boxShadow${string}`>;
```

Comment:
If you mean to expose this BoxShadowsType to users it should also be reexported in `index.ts` file

Reply:
Don't think it needs to be exposed so have removed the export